london 1888/__tests__/london1888.tests.js

```diff
@@ -0,0 +1,74 @@
+import {BASIC_CITIZEN, CLOSEST_CITIZEN, OTHER_BASIC_CITIZEN, SAME_CLOSEST_CITIZEN, VICTIM} from "../mocks/citizen.mock";
+import {getLondon1888} from "../london1888";
+import ConflictError from "../errors/conflict.error";
+import NotfoundError from "../errors/notfound.error";
+
+const mockGetAllCitizensAsync = jest.fn();
+const mockGetVictimAsync = jest.fn();
+
+jest.mock('../dal.js', () => {
+    return jest.fn().mockImplementation(() => ({
+        getAllCitizenAsync: mockGetAllCitizensAsync,
+        getVictimAsync: mockGetVictimAsync,
+    }))
+})
+
+describe('London 1888', () => {
+    it('Get all citizen : Done',async () => {
```

Comment:
```suggestion
    it('Get all citizen : Done', async () => {
```
Pas d'espce sur toutes les méthodes 🙄